bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml

```diff
@@ -183,14 +210,14 @@ limitations under the License.
                   <exclude>org.slf4j:slf4j-log4j12</exclude>
                   <exclude>log4j:log4j</exclude>
                   <exclude>org.apache.htrace:htrace-core</exclude>
-                  <exclude>
-                    com.github.stephenc.findbugs:findbugs-annotations
-                  </exclude>
-                  <exclude>log4j:log4j</exclude>
```

Comment:
why did you remove the exclude for log4j?

Reply:
duplicate. see lime 183/211